tests/ml_test.py

```diff
@@ -86,6 +94,30 @@ def test_output_values_missing(self):
             )
         )
 
+    def test_tree_predict(self):
+        with gw.config.update(
+            ref_res=300,
+        ):
+            with gw.open(l8_224078_20200518, nodata=0) as src:
+                with warnings.catch_warnings():
+                    warnings.simplefilter(
+                        "ignore",
+                        (DeprecationWarning, FutureWarning, UserWarning),
+                    )
+                    X, Xy, clf = fit(src, tree_pipeline, aoi_poly, col="lc")
+                    y1 = predict(src, X, clf)
+                    y2 = fit_predict(src, tree_pipeline, aoi_poly, col="lc")
+                    
+        self.assertTrue(np.all(np.isnan(y1.values[0, 0:5, 0])))
+        self.assertTrue(np.all(np.isnan(y2.values[0, 0:5, 0])))
+        self.assertTrue(
+            np.allclose(
+                y1.values[0, -5:-1, 0],
```

Comment:
Curious what's going on here with the indexing?

Reply:
Just grabbing some portion of the values to compare. I guess I really don't need to do that but oh well.